cmd/endpoint.go

```diff
@@ -308,6 +308,8 @@ func CreateEndpoints(serverAddr string, args ...string) (string, EndpointList, S
 					err = fmt.Errorf("path '%s' can not be served from different address/port", endpoint.Path)
```

Comment:
This message should say `path '%s' cannot be served by different Minio instances on the same server` - otherwise the meaning is opposite of what you want to say.